src/table/cells/__tests__/withStyling.spec.jsx

```diff
@@ -0,0 +1,32 @@
+import '../../__tests__/rtl-setup';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import * as withStyling from '../withStyling';
+
+describe('withStyling', () => {
+  const sandbox = sinon.createSandbox();
+  let HOC;
+  let styling;
+
+  beforeEach(() => {
+    // This is a bit of dummy mock. value will be on props, but this is not how the value is set. That is done in tableBodyWrapper
+    // But this enables testing that withStyling uses the CellComponent correctly
+    HOC = withStyling.default((props) => <div {...props}>{props.value}</div>);
```

Comment:
Not the best test strategy... The thing is that checking if an element has a certain class using RTL is bad practice. You should only look at what the user could see. So all we don here is pass this mocked div through `withStyling` and check if it comes out the other side. For `withColumnStyling` we at least test if `getColumnStyle` is called properly, so that test makes sense. I'm 50/50 about keeping this, doesn't test anything properly, but keeps the coverage up 🤷